examples/index.js

```diff
@@ -26,71 +26,114 @@ const hawkularAPM = require('../index');
 const SERVER_PORT = 9000;
 
 opentracing.initGlobalTracer(new hawkularAPM.APMTracer({
-    recorder: new hawkularAPM.ConsoleRecorder(),
+    recorder: new hawkularAPM.HttpRecorder('http://localhost:8080', 'jdoe', 'password'),
+    // recorder: new hawkularAPM.ConsoleRecorder(),
     sampler: new hawkularAPM.AlwaysSampledSampler(),
 }));
 
 // /hello handler
 dispatcher.onGet('/hello', function(req, res) {
     const serverSpan = opentracing.globalTracer().startSpan('hello', {
+        childOf: extractSpanContext(opentracing.globalTracer(), req.headers),
         tags: {
+            'http.method': 'GET',
             'http.url': extractUrl(req),
             'http.status_code': 200,
         }
     });
+    serverSpan.log({
+        size: 25,
+    });
+
     res.writeHead(200);
     res.end('Hello from Node.js!');
+    serverSpan.setTag('http.status_code', 200);
     serverSpan.finish();
 });
 
 // /a handler
 dispatcher.onGet("/a", function(req, res) {
     const serverSpan = opentracing.globalTracer().startSpan('/a', {
+        childOf: extractSpanContext(opentracing.globalTracer(), req.headers),
         tags: {
+            'http.method': 'GET',
             'http.url': extractUrl(req),
-            'http.status_code': 200,
         }
     });
 
-    const clientSpan = opentracing.globalTracer().startSpan('/b', {
+    const clientSpanB = opentracing.globalTracer().startSpan('/b', {
         childOf: serverSpan,
         tags: {
-            'http.url': extractUrl(req),
-            'http.status_code': 200,
+            'http.method': 'GET',
+            'http.url': 'http://localhost:' + SERVER_PORT + '/b',
         }
     });
     http.get({
         host: 'localhost',
         port: SERVER_PORT,
         path: '/b',
-        headers: createCarrier(opentracing.globalTracer(), clientSpan),
+        headers: createCarrier(opentracing.globalTracer(), clientSpanB),
     }, function (response) {
-        clientSpan.finish();
+        clientSpanB.setTag('http.status_code', 200);
+        clientSpanB.finish();
+    });
+
+    const clientSpanC = opentracing.globalTracer().startSpan('/c', {
+        childOf: serverSpan,
+        tags: {
+            'http.method': 'GET',
+            'http.url': 'http://localhost:' + SERVER_PORT + '/c',
+        }
+    });
+    http.get({
+        host: 'localhost',
+        port: SERVER_PORT,
+        path: '/c',
+        headers: createCarrier(opentracing.globalTracer(), clientSpanC),
+    }, function (response) {
+        clientSpanC.setTag('http.status_code', 200);
+        clientSpanC.finish();
     });
 
     res.writeHead(200);
-    res.end('/a');
+    res.end('a operation');
+    serverSpan.setTag('http.status_code', 200);
     serverSpan.finish();
```

Comment:
+1 - yes just re-read the opentracing description and it is valid. I think it would be good to add a comment on the /a span finish just to make it clear that it is completing before the child spans?